src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +16,71 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (PersistenceException persistenceException) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Couldn't save comment: "
+                    + entity, persistenceException);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return entity;
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        Comment comment;
+        try (Session session = factory.openSession()) {
+            comment = session.get(Comment.class, id);
+        }
+        return comment;
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        List<Comment> allComments;
+        try (Session session = factory.openSession()) {
+            Query<Comment> allCommentsQuery = session
+                    .createQuery("from Comment", Comment.class);
+            allComments = allCommentsQuery.getResultList();
+        } catch (PersistenceException persistenceException) {
+            throw new RuntimeException("Couldn't get all comments from DB",
+                    persistenceException);
+        }
+
+        return allComments;
```

Comment:
```suggestion
        List<Comment> allComments;
        try (Session session = factory.openSession()) {
            Query<Comment> allCommentsQuery = session
                    .createQuery("from Comment", Comment.class);
            allComments = allCommentsQuery.getResultList();
        } catch (PersistenceException persistenceException) {
            throw new RuntimeException("Couldn't get all comments from DB",
                    persistenceException);
        }

        return allComments;
```

why do you catch exception in getAll method, but dont catch it in get method? Lets make our code more consistent  and catch exceptions everywhere or nowhere